tests/std/tests/P0896R4_views_take/test.cpp

```diff
@@ -561,22 +561,17 @@ constexpr void move_only_test() {
 }
 
 constexpr void output_range_test() {
-#if !defined(__clang__) && !defined(__EDG__) // TRANSITION, VSO-1132704
-    if (!is_constant_evaluated())
-#endif // ^^^ workaround ^^^
-    {
-        using R = test::range<output_iterator_tag, int, test::Sized::no, test::CanDifference::no, test::Common::no,
-            test::CanCompare::no, test::ProxyRef::yes, test::CanView::yes, test::Copyability::move_only>;
-        int some_writable_ints[] = {0, 1, 2, 3};
-        static_assert(same_as<decltype(views::take(R{some_writable_ints}, 99999)), ranges::take_view<R>>);
+    using R = test::range<output_iterator_tag, int, test::Sized::no, test::CanDifference::no, test::Common::no,
+        test::CanCompare::no, test::ProxyRef::yes, test::CanView::yes, test::Copyability::move_only>;
+    int some_writable_ints[] = {0, 1, 2, 3};
+    static_assert(same_as<decltype(views::take(R{some_writable_ints}, 99999)), ranges::take_view<R>>);
 
-        // How do I implement "Fill up to n elements in {output range} with {value}"?
-        ranges::fill(R{some_writable_ints} | views::take(99999), 42);
-        assert(ranges::equal(some_writable_ints, initializer_list<int>{42, 42, 42, 42}));
+    // How do I implement "Fill up to n elements in {output range} with {value}"?

```

Comment:
out of scope - what's the context behind this comment? Isn't the line below doing precisely this (filling up to `99999` elements with `42`)?

Reply:
I'm not sure - this goes back to the original PR #1253 on 2020-09-04. I think it's a rhetorical question, to be immediately answered by the code below, but it's not our usual style.